ide-migration/server/migration/api/migration-service.js

```diff
@@ -365,6 +354,48 @@ class MigrationService {
         hdiFile.setText(hdiJson);
     }
 
+    addFilesWithoutGenerated(userData, workspace, locals) {
+        for (const local of locals) {
+            this.addFileToWorkspace(workspace, local.repositoryPath, local.relativePath, local.projectName);
+            const projectName = local.projectName;
+            let repos = git.getGitRepositories(workspace);
+            let repoExists = false;
+            for (const repo of repos) {
+                if (repo.getName() === projectName) {
+                    repoExists = true;
+                    break;
+                }
+            }
+            if (repoExists) {
+                git.commit('migration', '', userData.workspace, projectName, 'Overwrite existing project', true);
+            } else {
+                console.log("Initializing repository...")
+                git.initRepository('migration', '', workspace, projectName, projectName, "Migration initial commit");
+            }
+        }
+    }
+
+    addGeneratedFiles(userData, deliveryUnit, workspace, locals) {
+        for (const local of locals) {
+            const projectName = local.projectName;
+            const generated = deliveryUnit['deployableArtifactsResult']['generated'].filter(x => x.projectName === projectName);
+            for (const gen of generated) {
```

Comment:
Could you rename `generated` to `generatedFiles` and `gen` to `generatedFile`?

Reply:
+1